probedata.go

```diff
@@ -24,8 +24,20 @@ const (
 
 // ProbeData is the root json data structure returned by an ffprobe.
 type ProbeData struct {
```

Comment:
It's probably nice to add a couple of util methods to the struct for accessing chapters